localdev/seed/api.py

```diff
@@ -203,8 +203,38 @@ def _delete(self, model_cls, data):
         self.seed_result.add_deleted(deleted_type_counts)
         return deleted_count, deleted_type_counts
 
+    def _create_cms_resource_page(self, resource_page):
+        """ Create resource pages in CMS if they don't exists."""
+        existing_obj = ResourcePage.objects.filter(slug=resource_page["slug"]).first()
+        if existing_obj:
+            self.seed_result.add_existing(existing_obj)
+            return existing_obj
```

Comment:
are we using that `existing_obj` in the method declaring point ? if not then simply `return`